pkgs/development/python-modules/aiomultiprocess/default.nix

```diff
@@ -1,25 +1,27 @@
-{ lib
-, buildPythonPackage
-, fetchFromGitHub
-, flit-core
-, pytestCheckHook
-, pythonOlder
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  pytestCheckHook,
+  pythonOlder,
```

Comment:
You shouldn't change the format of the arguments; it's [Nixpkgs standard convention](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax).

Reply:
This is the new format that comes with RFC 0166.